ttnn/cpp/ttnn/operations/moreh/moreh_group_norm/moreh_group_norm.hpp

```diff
@@ -0,0 +1,31 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+#include "ttnn/decorators.hpp"
+#include "ttnn/operations/core/compute_kernel/compute_kernel_config.hpp"
+
+namespace ttnn::operations::moreh::moreh_group_norm {
+struct MorehGroupNorm {
+    static std::vector<std::optional<Tensor>> invoke(
```

Comment:
Why does this return vector<optional<Tensor>> and not Tensor?

Reply:
The output_tensors includes 3 tensors (output, mean, and rtsd), 2 of those (mean, and rtsd) are optional